### DISCUSSION TITLE
Request for help concerning your freemocap project

### DISCUSSION THREAD

**Discussion by TassaditHAMMOUNI** [freemocap/freemocap#94] [General]
Hello
I would like to ask you to help me regarding your freemocap program because I am working on it and I am trying to run it shows me errors in the calibration step.

**Comment by abitrolly** [comment]
Better post what your setup is, what you're doing and your errors here. Then more people will be able to help.

**Comment by TassaditHAMMOUNI** [comment]
[
[image](https://user-images.githubusercontent.com/97808739/149659573-7b3a0186-ac78-4e77-b6dc-20e85e0e4148.png)
](url)

**Reply by TassaditHAMMOUNI** [reply]
here is the error it shows me
[
![image](https://user-images.githubusercontent.com/97808739/149659629-ac4864ee-bd7a-40c3-a171-44e6cf3f3df3.png)
](url)

**Reply by abitrolly** [reply]
That's tracked here https://github.com/jonmatthis/freemocap/issues/95
